core/src/main/java/com/google/instrumentation/trace/Span.java

```diff
@@ -73,53 +73,60 @@ protected Span(SpanContext context, @Nullable EnumSet<Options> options) {
   /**
    * Adds a set of attributes to the {@code Span}.
    *
-   * @param attributes The attributes that will be added and associated with the {@code Span}.
+   * @param attributes the attributes that will be added and associated with the {@code Span}.
    */
   public abstract void addAttributes(Attributes attributes);
 
   /**
    * Adds an annotation to the {@code Span}.
    *
-   * @param description The description of the annotation time event.
+   * @param description the description of the annotation time event.
    */
   public abstract void addAnnotation(String description);
```

Comment:
is this the same as attribute with a description and empty attributes? If so, you could make this a concrete, but overridable method.